driver/Makefile

```diff
@@ -1,8 +1,8 @@
 CONFIG_MODULE_SIG = n
 Q                := @
 PWD              ?= $(shell pwd)
-KVERSION         ?= $(shell uname -r)
-KERNEL_SRC       ?= /lib/modules/$(KVERSION)/build
+KVERSION         ?= $(ls /lib/modules | tail -n 1)
+KERNEL_SRC       ?= $(DKMS_KERNEL_SOURCE_DIR)
```

Comment:
Build for the installed kernel sources instead of the reported by uname -r.
This resolves problems on containers and in the mowers we always have a fixed version of the kernel.